src/interfaces/source.ts

```diff
@@ -286,14 +287,14 @@ export class ModLockPost {
   when_: string;
 }
 
-export class ModStickyPost {
+export class ModFeaturePost {
   id: number;
   mod_person_id: number;
   post_id: number;
   @Transform(({ value }) => toOption(value), { toClassOnly: true })
   @Transform(({ value }) => toUndefined(value), { toPlainOnly: true })
   @Expose()
-  stickied: Option<boolean>;
+  featured: Option<boolean>;
```

Comment:
Hrm.. this doesn't seem to make much sense for it to be optional. Better check the back end. Also I'll wait on merging cause the back-end still needs the local / community differentiation.

Reply:
I was thinking the same thing, wasn't sure why it was optional. Ill change it in that case.

Reply:
Once that's ready to go, I can merge and you should be able to fix the federation tests for https://github.com/LemmyNet/lemmy/pull/2585

Reply:
Done. Forgot to push it, loll.